src/runtime_src/core/include/xclbin.h

```diff
@@ -196,6 +196,7 @@ extern "C" {
         IP_MEM_DDR4,
         IP_MEM_HBM,
         IP_MEM_HBM_ECC,
+        IP_PS_KERNEL,
```

Comment:
Hmm....seems you are 1 step ahead of me :^).

I was going to ask you about this next week, but since have this PR.  

Q: Will this enumeration value be used for both the "Soft" and "Fixed" PS Kernels?

Reply:
Yes, I imagine for both.